### DISCUSSION TITLE
extract::State

### DISCUSSION THREAD

**Discussion by ishtms** [tokio-rs/axum#1273] [Q&A] [answered]
When will a new version be pushed, that includes the recent addition of the `State` extractor?

**Comment by jplatte** [comment]
Adding the `State` executor required making breaking changes, so it will only come out with version 0.6. Since this is not a commercial project, we do not provide ETAs, but you can have a look at the [0.6 milestone](https://github.com/tokio-rs/axum/milestone/5).

**Comment by davidpdrsn** [comment, marked as answer]
Don't know yet but you can follow along with the progress in [the milestone.](https://github.com/tokio-rs/axum/milestone/5)
